kafka-connect/build.gradle

```diff
@@ -30,3 +30,30 @@ project(":iceberg-kafka-connect:iceberg-kafka-connect-events") {
     useJUnitPlatform()
   }  
 }
+
+project(":iceberg-kafka-connect:iceberg-kafka-connect") {
```

Comment:
Why the duplicate name? Could this be `:iceberg-kafka-connect:iceberg-sink`?

Reply:
I was following the convention of the `spark` projects to some extent, e.g. `:iceberg-spark:iceberg-spark-<version>`